### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/customize/external.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#855] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/customize/external.html

本教程将指导你如何将外部脚本和样式添加到你的站点。

https://theme-hope.vuejs.press/zh/guide/customize/external.html

**Comment by bihell** [comment]
/* your style here */ 

格式要怎么写？ 我想把某页弄成黑白的，直接像下面这样不行。。。

---
head:
  - style:
      type: text/css
      content: |
          html {
            -webkit-filter: grayscale(100%);
            -moz-filter: grayscale(100%);
            -ms-filter: grayscale(100%);
            -o-filter: grayscale(100%);
            filter: progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
            _filter: none;
         }
---

**Reply by Mister-Hope** [reply]
格式错了，学点yaml吧 ❛‿˂̵✧
